docs/gettingstarted/index.rst

```diff
@@ -133,7 +133,7 @@ Getting started with Flocker
 
          .. code-block:: console
 
-            you@laptop:~$ sudo apt-get update && apt-get install -y gcc python2.7 python-virtualenv python2.7-dev && \
+            you@laptop:~$ sudo apt-get update && sudo apt-get install -y gcc libffi-dev libssl-dev python2.7 python-virtualenv python2.7-dev && \
```

Comment:
Acutually, the extra libraries don't appear to be needed. I just spun up a fresh ubuntu machine, and managed to follow the instructions without installing these two libraries.